addons/website/static/src/client_actions/configurator/configurator.js

```diff
@@ -215,6 +217,9 @@ class DescriptionScreen extends Component {
                 this.labelToId[r.label] = r.id;
             });
         } else {
+            if (terms[0].length >= this.sendIap.length) {
+                this.sendIap = terms[0];
```

Comment:
`sendIap` is not a good name, it's impossible to figure what it holds.
Maybe `unmatchTerm` ? notFoundTerm, unexistingTerm, unmatchTerm, unavailableTerm

And probably rename the python key the same: `unmatch_term` instead of `attempt`, otherwise it's too complicated to understand / figure when key has different unrelated name between the js part and the python part